cmd/log.go

```diff
@@ -80,29 +111,70 @@ func (o *LogOptions) Run() error {
 	// convert Page and PageSize to Limit and Offset
 	page := util.NewPage(o.Page, o.PageSize)
 
-	p := &lib.LogParams{
-		Ref: o.Refs.Ref(),
-		ListParams: lib.ListParams{
+	for _, ref := range o.Refs.RefList() {
```

Comment:
Just tried this to show multiple logs
```
qri log yashagarwal/all_week qri_eng/qri_cli_user_benchmarks
```
didn't work because max args is 1. I vote we stick with that and don't bother with the loop. listing multiple things at once is confusing.